src/components/DataController/DataController.js

```diff
@@ -87,6 +95,7 @@ var DataController = createReactClass({
       this.loadAnnualCycleGraph(props);
       this.loadLongTermAverageGraph(props);
       this.loadDataTable(props);
+      this.loadContextGraph(props);
```

Comment:
These method names are confusing in several ways:

- they don't load a graph, they load the data for a graph
- they set state with an object called "graph", but the state property name is "xxxData"
- "graph" here actually means an object containing data and C3 options that is used (via DataGraph to render a C3 graph)

Strongly suggest that identifiers are precise about what they do or contain, and do not use confusing abridgements of meaning as here. This occurs in many places in this code. Kindly rename these so that their names reflect their use and meaning.